acceptance/bundle/variables/complex-transitive-deep/output.txt

```diff
@@ -1,3 +1,3 @@
 {
-  "spark.databricks.sql.initial.catalog.name": "${var.catalog}"
```

Comment:
Can you add a test for cases when there are nested cycle dependencies?

Reply:
See acceptance/bundle/variables/complex-cycle/databricks.yml